tests/driver_si70xx/Makefile

```diff
@@ -9,8 +9,8 @@ USEMODULE += xtimer
 
 # set default device parameters in case they are undefined
 TEST_I2C ?= 0
-TEST_I2C_ADDR ?= 128
-TEST_PIN_EN ?= 57
+TEST_I2C_ADDR ?= 0x80
+TEST_PIN_EN ?= GPIO_PIN\(PD,9\)
```

Comment:
I think it would make sense to choose something more generic here, as `GPIO_PIN(0, 0)`, as `PD` is not defined for every cpu...